cmd/webhook/main.go

```diff
@@ -85,7 +85,7 @@ func main() {
 	})
 
 	sharedmain.WebhookMainWithContext(
-		ctx, "net-istio-webhook",
+		sharedmain.WithHealthProbesDisabled(ctx), "net-istio-webhook",
```

Comment:
Deactivating the default endpooint as we will use the webhook one.

Reply:
Although it makes sense, do you know why serving's webhook is not disabling it ? - https://github.com/knative/serving/blob/main/cmd/webhook/main.go#L166-L171

Reply:
I added the default probes at the knative/pkg side and it was suggested to enable by default :shrug:. I guess I can do a follow up there too.